dashboards-observability/public/components/visualizations/charts/stats/stats.tsx

```diff
@@ -129,10 +127,11 @@ export const Stats = ({ visualizations, layout, config }: any) => {
     if (metricLength === 1) {
       return 0.5;
     } else {
+      // calculate center of respective axis in each subplot based on metric length
       if (index === 0) {
-        return ((1 / metricLength) * 1) / 2;
+        return 1 / metricLength / 2;
```

Comment:
Could you add a little bit more elaborations on your algorithms for calculting center of respective axis in each subplot? May help others to quickly grab the way it does the calculation. thanks